src/test/test-unit-name.c

```diff
@@ -463,7 +465,14 @@ static void test_unit_name_path_unescape(void) {
 }
 
 int main(int argc, char* argv[]) {
+        _cleanup_(rm_rf_physical_and_freep) char *runtime_dir = NULL;
         int rc = 0;
+
+        log_parse_environment();
+        log_open();
+
+        assert_se(runtime_dir = setup_fake_runtime_dir());
```

Comment:
`test_unit_printf` is still using `assert_se(setenv("XDG_RUNTIME_DIR", "/run/user/1/", 1) == 0)`, which doesn't seem to be needed any more.